### PR TITLE
Working BFS code for multiple hosts

### PR DESCRIPTION
**Pull request by anubhavgoel26**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -70,7 +70,8 @@ set(PCG_DIRECTORY ${Galois_SOURCE_DIR}/external/pcg-cpp/include)
 
 add_subdirectory(microbench)
 add_subdirectory(scripts)
-add_subdirectory(wf4)
```
Comment (patrickkenney9801) [inline CMakeLists.txt:73]: was wf4 broken by the changes or was this an accident?

Comment (anubhavgoel26) [reply CMakeLists.txt:73]: @divija95 did you remove wf4 on purpose?

Comment (divija95) [reply CMakeLists.txt:73]: Yes, wf4 has some incompatible apis wrt DistLocalGraph
ToDO: @divija95 to fix it

**Comment by nicelhc13** [comment]
lemme first merge it to PortingBFS for tests.

**Pull request merged**